metrics: Update apple system font metrics (#193)
seek-oss/capsize: .changeset/curvy-forks-brush.md

---
'@capsizecss/metrics': patch
---

metrics: Update apple system font metrics

Previously the metrics provided for `-apple-system` and `BlinkMacSystemFont` were extracted from the `SF Pro` font, with a custom override to correct the `descent` metric.

> Through upcoming work to support font weights and styles, it has been identified that MacOS uses the `SFNS` font.

Reviewer: ```suggestion
During upcoming work to support font weights and styles, it was identified that MacOS uses the `SFNS` font.
```

I don't know why I struggled to read this line, but I did and wanted to suggest something.

It might just be me though, so just resolve if you disagree.

Author: Hmm a funny one, as reading yours makes me question how i use English lol. There is a lot of tense bouncing around. Thoughts on this:

```suggestion
Through work to support metrics for different font weights and styles, it was identified that MacOS uses the `SFNS` font.
```

Reviewer: It's fine. I think I just struggle with sentences starting with "Through". But I think that's my poor english skills showing.
